bounties/BA122402.md

```diff
@@ -0,0 +1,47 @@
+---
+code: BA122402
+title: Smart Node UI Wireframes
+proposal-author: Patches
+creation-date: 2024-05-28
+external-status: open
+summary: Design wireframes that could be used by a future UI developer and graphic designer to create a front-end for Smart Node.
```

Comment:
```suggestion
summary: Design wireframes that could be used by a future UI developer and graphic designer to create a front-end for the Rocket Pool Smart Node Package.
```
Name from the github? <https://github.com/rocket-pool/smartnode?tab=readme-ov-file#rocket-pool---smart-node-package>. I know its known more colloquially as smartnode by those in the community.